### ISSUE TITLE
Style/SpaceAfterNot doesn't catch not then parenthesis: ! (foo)

### ISSUE THREAD

**Issue by cgriego** [bbatsov/rubocop#3043]
## Expected behavior

This code should have reported an offense for the Style/SpaceAfterNot cop:
```ruby
@accept_success = ! (model.send(association).empty?)
```

## Actual behavior

No offense was reported. After I ran and auto-corrected Style/RedundantParentheses, then an offense was reported for the auto-corrected code.

```ruby
@accept_success = ! model.send(association).empty?
```

## RuboCop version

0.39.0 (using Parser 2.3.0.7, running on ruby 1.9.3 x86_64-darwin14.0.0)

**Comment by rrosenblum** [comment]
This should be an easy fix. Reducing down the problem, `! (model)` does not register an offense either. I see that the cop is using regex, so that probably needs to be tweaked. I should be able to have a fix for this soon.
